core/server/services/members/config.js

```diff
@@ -38,8 +38,11 @@ class MembersConfigProvider {
      */
     getEmailFromAddress() {
         const subscriptionSettings = this._settingsCache.get('members_subscription_settings') || {};
-
-        return `${subscriptionSettings.fromAddress || 'noreply'}@${this._getDomain()}`;
+        // CASE: Default settings json has fromAddress as `noreply` which we want to handle
+        if (!subscriptionSettings.fromAddress || subscriptionSettings.fromAddress === 'noreply') {
+            return `noreply@${this._getDomain()}`;
+        }
+        return subscriptionSettings.fromAddress;
```

Comment:
The main reason `noreply` needs special handling atm is because of default settings [here](https://github.com/TryGhost/Ghost/blob/1d49cba426dea8d54edeb6ab2ba78b4ab76c821c/core/server/data/schema/default-settings.json#L219) which inserts `noreply` as the default `fromAddress`. This probably needs a better way to be initialized as part of default settings in longer term 🤔